spec/features/multiple_emails/sp_sign_in_spec.rb

```diff
@@ -5,45 +5,44 @@
   include OidcAuthHelper
 
   context 'with the email scope' do
-    scenario 'signing in with OIDC sends the email address used to sign in' do
+    scenario 'signing in with OIDC sends the user last email address' do
       user = create(:user, :fully_registered, :with_multiple_emails)
-      emails = user.reload.email_addresses.map(&:email)
+      emails = user.reload.email_addresses
 
       expect(emails.count).to eq(2)
-
       emails.each do |email|
         visit_idp_from_oidc_sp(scope: 'openid email')
-        signin(email, user.password)
+        signin(email.email, user.password)
         fill_in_code_with_last_phone_otp
         click_submit_default
         click_agree_and_continue if current_path == sign_up_completed_path
-
+        identity_email_address_id = user.identities.last.email_address_id
         decoded_id_token = fetch_oidc_id_token_info
```

Comment:
I'd like to see how I can get the email address that is being sent to the SP

Reply:
test added at https://github.com/18F/identity-idp/pull/10951/commits/3ebac6a0c87b7be3cdcb458dd394da819ce2eaf0